python/MaterialXTest/shaderx.py

```diff
@@ -0,0 +1,118 @@
+import math
+import os
+import unittest
+
+import MaterialX as mx
+
+_fileDir = os.path.dirname(os.path.abspath(__file__))
+
+def _getSubDirectories(libraryPath):
```

Comment:
I guess this was due to the fact there is no FilePath Pythong wrappers. Maybe ask about this to find out if this was an oversight or on purpose.  As mentioned I don't think we want loadLibraries (plural) is always the correct logic, but maybe  getSubDirs, getMtxFiles and loadLibrary may be.

Reply:
As discussed will look into this in a future review.